recipes-bsp/optee-ftpm/optee-ftpm.bb

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Su Bao Cheng <baocheng.su@siemens.com>
+#
+# SPDX-License-Identifier: MIT
+#
+inherit dpkg
+
+SUMMARY = "OPTEE fTPM Microsoft TA"
+DESCRIPTION = "TCG reference implementation of the TPM 2.0 Specification."
+HOMEPAGE = "https://github.com/microsoft/ms-tpm-20-ref/"
+
+FTPM_UUID="bc50d971-d4c9-42c4-82cb-343fb7f37896"
```

Comment:
Should this be a default assignment.

Reply:
This UUID is defined in https://github.com/microsoft/ms-tpm-20-ref/blob/main/Samples/ARM32-FirmwareTPM/optee_ta/fTPM/include/fTPM.h